### DISCUSSION TITLE
no latin5/iso8859-9 charset support

### DISCUSSION THREAD

**Discussion by ozgyilmaz** [scandum/tintin#119] [Ideas]
hello,

tintin doesnt support turkish charsets latin5/iso8859-9. is there a plan to implement it?

**Comment by scandum** [comment]
You can use `#config charset ascii` to disable UTF-8 and things should work if the terminal supports Turkish.

If you want iso8859-9 automatically translated to unicode that's not something that's planned for implementation.

**Comment by mangusmud** [comment]
I think  tintin++ considersTurkish characters in ASCII map as non-printable. I've tried to upload an attachment, hope it'll show how Turkish chars are printed although my linux shell's locale is ISO8859-9.

![tintin](https://user-images.githubusercontent.com/112348373/188941366-f90b9bd3-ce9a-4c89-a9aa-d2b44aed3180.png)

**Reply by scandum** [reply]
Looks to me like your terminal is set to Unicode, see if you can change it to ISO8859-9 as well?
